tests/test_changelog.py

```diff
@@ -242,6 +242,21 @@ def test_entries_dir_not_created(self, mock_project):
             # Nothing was created in the directory
             self.assertEqual(len(os.listdir(fs)), 0)
 
+    def test_validate_good_entry(self):
```

Comment:
As you are testing both cases, rename this function to `test_validate_entry`

Reply:
We also need a test checking the whole process. Now that you have `validate_changelog_content` as a function, you can mock it and check what happens when it returns either `True` or `False`.